source/settings/command-line-options.txt

```diff
@@ -138,23 +157,54 @@ value in the :guilabel:`Settings` panel.
 
        To learn more, see :ref:`compass-configure-network-traffic`.
 
+   * - .. option:: --password
+     - Specifies a password for authentication to Compass and the MongoDB 
+       deployment provided in the connection string. 
+
+       Use in conjunction with :option:`--username`. 
+
+   * - .. option:: --persistOIDCTokens
+     - Remain logged in when using the MongoDB OIDC authentication mechanism for 
+       MongoDB server connections. Session tokens are encrypted and stored using 
+       the system keychain.
+
    * - .. option:: --protectConnectionStrings
      - .. include:: /includes/fact-protect-connection-strings.rst  
    
        To learn more, see :ref:`compass-protect-connection-strings`.
 
+   * - .. option:: --protectConnectionStringsForNewConnections
+     - Sets connection strings for new connections as read-only by
+       default. Passwords in new connection strings display as
+       ``*****``.
+
+       If ``protectConnctionStringsForNewConnections`` is enabled,
+       |compass-short| disables the :guilabel:`Edit connection string`
+       option but doesn't prevent users from manually enabling the 
+       option with the :guilabel:`Edit connection string` toggle. 
+
    * - .. option:: --readOnly
      - Prevent users from performing write operations to your MongoDB deployment 
        through |compass-short|.
 
        To learn more, see :ref:`compass-read-only`.
 
+   * - .. option:: --showInsights
+     - Enables performance insights for your schemas and queries. 
+
+       To learn more, see :ref:`compass-performance-insights`.
+
    * - .. option:: --showKerberosPasswordField
      - Show or hide the Kerberos password field on the |compass-short| 
        connection form.
 
        To learn more, see :ref:`compass-kerberos-password`.
 
+   * - .. option:: --showOIDCDeviceAuthFlow
+     - Enable device authorization flow for OIDC authentication on MongoDB server. 
```

Comment:
[suggestion] small suggestion for consistency!

```suggestion
     - Enables device authorization flow for OIDC authentication on MongoDB server. 
```

Reply:
Updated.